src/protocolsupport/zplatform/impl/pe/PEProxyServerInfoHandler.java

```diff
@@ -30,6 +31,7 @@
 
 	private static final int statusThreads = JavaSystemProperty.getValue("pestatusthreads", 2, Integer::parseInt);
 	private static final int statusThreadKeepAlive = JavaSystemProperty.getValue("pestatusthreadskeepalive", 60, Integer::parseInt);
+	private static final int randomId = new Random().nextInt();
```

Comment:
Use ThreadLocalRandom. And it should be a string from a start to save time on String.valueOf calls

Reply:
I hardly believe this matters. We're doing repeated regexp operations on the PE version string... If we were worried about extreme performance in this method, we should start there. And it's very much unclear that creating a thread local random PRNG for a single call at startup really isn't a regression, rather than an optimization.

Reply:
We don't do regex on pe version tho, or at least not supposed to. So it does matter even if it's a simple thing. Also you don't save anything on not calling threadlocalrandom, because the seed for it is actually stored in the thread.